checks.d/mysql.py

```diff
@@ -32,78 +33,341 @@
+    'Innodb_data_pending_fsyncs': ('mysql.innodb.data_pending_fsyncs', GAUGE),
+    'Innodb_data_pending_reads': ('mysql.innodb.data_pending_reads', GAUGE),
+    'Innodb_data_pending_writes': ('mysql.innodb.data_pending_writes', GAUGE),
+    'Innodb_data_read': ('mysql.innodb.data_read', GAUGE),
+    'Innodb_data_written': ('mysql.innodb.data_written', GAUGE),
+    'Innodb_dblwr_pages_written': ('mysql.innodb.dblwr_pages_written', GAUGE),
+    'Innodb_dblwr_writes': ('mysql.innodb.dblwr_writes', GAUGE),
+    'Innodb_hash_index_cells_total': ('mysql.innodb.hash_index_cells_total', GAUGE),
+    'Innodb_hash_index_cells_used': ('mysql.innodb.hash_index_cells_used', GAUGE),
+    'Innodb_history_list_length': ('mysql.innodb.history_list_length', GAUGE),
+    'Innodb_ibuf_free_list': ('mysql.innodb.ibuf_free_list', GAUGE),
+    'Innodb_ibuf_merged': ('mysql.innodb.ibuf_merged', RATE),
+    'Innodb_ibuf_merged_delete_marks': ('mysql.innodb.ibuf_merged_delete_marks', RATE),
+    'Innodb_ibuf_merged_deletes': ('mysql.innodb.ibuf_merged_deletes', RATE),
+    'Innodb_ibuf_merged_inserts': ('mysql.innodb.ibuf_merged_inserts', RATE),
+    'Innodb_ibuf_merges': ('mysql.innodb.ibuf_merges', RATE),
+    'Innodb_ibuf_segment_size': ('mysql.innodb.ibuf_segment_size', GAUGE),
+    'Innodb_ibuf_size': ('mysql.innodb.ibuf_size', GAUGE),
+    'Innodb_lock_structs': ('mysql.innodb.lock_structs', RATE),
+    'Innodb_locked_tables': ('mysql.innodb.locked_tables', GAUGE),
+    'Innodb_locked_transactions': ('mysql.innodb.locked_transactions', GAUGE),
+    'Innodb_log_waits': ('mysql.innodb.log_waits', RATE),
+    'Innodb_log_write_requests': ('mysql.innodb.log_write_requests', RATE),
+    'Innodb_log_writes': ('mysql.innodb.log_writes', RATE),
+    'Innodb_lsn_current': ('mysql.innodb.lsn_current', RATE),
+    'Innodb_lsn_flushed': ('mysql.innodb.lsn_flushed', RATE),
+    'Innodb_lsn_last_checkpoint': ('mysql.innodb.lsn_last_checkpoint', RATE),
+    'Innodb_mem_adaptive_hash': ('mysql.innodb.mem_adaptive_hash', GAUGE),
+    'Innodb_mem_additional_pool': ('mysql.innodb.mem_additional_pool', GAUGE),
+    'Innodb_mem_dictionary': ('mysql.innodb.mem_dictionary', GAUGE),
+    'Innodb_mem_file_system': ('mysql.innodb.mem_file_system', GAUGE),
+    'Innodb_mem_lock_system': ('mysql.innodb.mem_lock_system', GAUGE),
+    'Innodb_mem_page_hash': ('mysql.innodb.mem_page_hash', GAUGE),
+    'Innodb_mem_recovery_system': ('mysql.innodb.mem_recovery_system', GAUGE),
+    'Innodb_mem_thread_hash': ('mysql.innodb.mem_thread_hash', GAUGE),
+    'Innodb_mem_total': ('mysql.innodb.mem_total', GAUGE),
+    'Innodb_os_file_fsyncs': ('mysql.innodb.os_file_fsyncs', RATE),
+    'Innodb_os_file_reads': ('mysql.innodb.os_file_reads', RATE),
+    'Innodb_os_file_writes': ('mysql.innodb.os_file_writes', RATE),
+    'Innodb_os_log_pending_fsyncs': ('mysql.innodb.os_log_pending_fsyncs', RATE),
+    'Innodb_os_log_pending_writes': ('mysql.innodb.os_log_pending_writes', RATE),
```

Comment:
`Innodb_os_log_pending_fsyncs` and `Innodb_os_log_pending_writes` are probably gauges (http://dev.mysql.com/doc/refman/5.7/en/server-status-variables.html#statvar_Innodb_os_log_pending_fsyncs)